File: Java/LootGen Server Side/LootGenCommons/src/main/java/joel/lajoiecorriveau/lootgen/models/Item_Chance.java
//package joel.lajoiecorriveau.lootgen.models;
package joel.lajoiecorriveau.lootgen.models;
/**
 * Created by Joel on 9/8/2015.
 */
public class Item_Chance {
    public Long ID;
    public Long ItemIndex;
    public Double dropPercent;

    public Item_Chance(Long pItemIndex, double pDropPercent)
    {
        ItemIndex = pItemIndex;
        dropPercent = pDropPercent;
    }


    public String ToString()
    {
        return Long.toString(ItemIndex) + "*|*" + Double.toString(dropPercent);
    }
}

File: C#/LootGen/LootGen/Item_Chance.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace LootGen
{
    public class Item_Chance
    {
        public int itemIndex;

        public decimal dropPercent;

        public Item_Chance(int pItemIndex, decimal pDropPercent)
        {
            itemIndex = pItemIndex;
            dropPercent = pDropPercent;
        }

        public override string ToString()
        {
            return itemIndex.ToString() + "*|*" + dropPercent.ToString();
        }
    }
}
